main.py

```diff
@@ -86,6 +102,61 @@
 
             # TODO
 
+            if not message_author in element.members:
+                element.members.append(message_author)
+
+            message = creation_messages.cc_intro(member_ids)
+            
+            # Role objects (based on ID)
+            main_guild = client.get_channel(bot_spam).guild # Find the guild we're in
+            roles = main_guild.roles # Roles from the guild
+            game_master_role = discord.utils.find(lambda r: r.id == game_master, roles) #WUSDIS                     # \
```

Comment:
What's with the `#WUSDIS` on this line and the following 2 lines? Get rid of it

Reply:
It means I don't know wus dis is.

Reply:
Get rid of it, there's comments

Reply:
Not on that one. What's utils? What's lambda? What's r.id?

Reply:
utils is a submodule of the discord library. It contains a `find()` function which finds things based on their objects. The lambda is comparing if `r.id` (the id of r, which is a role from `roles`) matches game_master